clang-ocl/PKGBUILD

```diff
@@ -1,14 +1,14 @@
 # Maintainer: Jakub Okoński <jakub@okonski.org>
 pkgname=clang-ocl
-pkgver=3.1.0
+pkgver=3.3.0
```

Comment:
Do you mind if do not upgrade clang-ocl? This package is slated to be removed, no need to continue development on it.

Reply:
Not at all, I did it first, that's why I included it here. I'm removing it.